meetings/__init__.py

```diff
@@ -95,37 +122,52 @@ def meeting_id(self, room_id):
     return f"{room_id}-{datetime.today().strftime('%Y-%m-%d')}"
     
   @command.new(aliases=["sm"])
-  async def startmeeting(self, evt: MessageEvent) -> None:
+  @command.argument("meetingname", pass_raw=True)
+  async def startmeeting(self, evt: MessageEvent, meetingname) -> None:
     meeting = await self.meeting_in_progress(evt.room_id)
-    
     if not await self.check_pl(evt):
-      await evt.respond("You do not have permission to start a meeting")
+      await self.send_respond(evt, "You do not have permission to start a meeting", meeting=meeting)
     elif meeting:
-      await evt.respond("Meeting already in progress")
+      await self.send_respond(evt, "Meeting already in progress", meeting=meeting)
     else:
       # Do backend-specific startmeeting things
       await self.backend.startmeeting(self, evt)
       
+      initial_topic = ""
+
+      if not meetingname:
+        roomname = await get_room_name(self.client, evt.room_id)
+        meetingname = f'{roomname} Meeting started {time_from_timestamp(evt.timestamp)}'
```

Comment:
This string is stored verbatim in the DB - does it not make sense to store *only* the the roomname if it is otherwise unset? I can see us wanting to re-use the meetingname in a variety of places, not "meetingname + bunch of other stuff"...